doc/sphinx/source/python_api/index.rst

```diff
@@ -8,9 +8,9 @@ This section contains nGraph™ Python API documentation. Python API exposes
 nGraph™ C++ operations to Python users. For quick-start you can find an example
 of the API usage below.
 
-.. literalinclude:: /../../../python/examples/basic.py
+.. literalinclude:: ../../../../python/examples/basic.py
```

Comment:
According to [doc](http://www.sphinx-doc.org/en/master/usage/restructuredtext/directives.html#directive-literalinclude): 
,, The file name is usually relative to the current file’s path. __However, if it is absolute (starting with /), it is relative to the top source directory.__ ''

It's unnecessary to add additional `..` to file path, however it's correct.

Reply:
Being consistent with the rest of the documentation is important